src/components/OcButton.vue

```diff
@@ -84,13 +84,21 @@ export default {
     },
     /**
      * Style variation to give additional meaning.
-     * `primary, danger`
+     * Defaults to `primary`.
+     * Can be `passive, primary, danger, success, warning`.
      */
     variation: {
       type: String,
-      default: "default",
+      default: "primary",
       validator: value => {
-        return value.match(/(default|primary|danger|raw)/)
+        return value.match(/(passive|primary|success|danger|warning)/)
+      },
+    },
+    appearance: {
```

Comment:
Docstring to keep it consistent. And developer-friendly 🙂

Reply:
Great catch, thanks!